Sources/StreamChatSwiftUI/ChatChannel/Composer/MessageComposerViewModel.swift

```diff
@@ -239,6 +239,10 @@ open class MessageComposerViewModel: ObservableObject {
         channelController.channel?.isDirectMessageChannel ?? false
     }
     
+    public var showCommandsOverlay: Bool {
+        composerCommand != nil && channelController.channel?.config.commands.count ?? 0 > 0
```

Comment:
I also have a tendency to write 1-liners, but I think this one is also not that "enjoyable" to read. Again, this is not something like "we NEED to change that", but I think it can be more readable. :)

Reply:
Fixed.